packages/ARU/PKGBUILD

```diff
@@ -0,0 +1,21 @@
+pkgname=kapitan
+pkgver=0.27.0
+pkgrel=1
+pkgdesc="Generic templated configuration management for Kubernetes, Terraform and other things"
```

Comment:
Same for the [description](https://github.com/deepmind/kapitan/blob/master/kapitan/version.py#L12)